src/plugins/dashboard/public/application/utils/get_no_items_message.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment } from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { EuiButton, EuiEmptyPrompt, EuiLink } from '@elastic/eui';
+import { ApplicationStart } from 'opensearch-dashboards/public';
+
+export const getNoItemsMessage = (
+  hideWriteControls: boolean,
+  createItem: () => void,
+  application: ApplicationStart
+) => {
+  if (hideWriteControls) {
+    return (
+      <div>
+        <EuiEmptyPrompt
+          iconType="visualizeApp"
```

Comment:
why a different icon?

Reply:
Will change.

Reply:
nice catch! this is actually a bug from the original dashboard: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/c5058a3ff2d46f52059d0d06a32bc93f300dd13d/src/plugins/dashboard/public/application/listing/dashboard_listing.js#L95 @joshuarrrr